develop/reference_sql.rst

```diff
@@ -26,6 +26,17 @@ one of three methods, in this order of preference:
    rewrite queries for *partial aggregation* on workers. For instance, to
    calculate an average, Citus obtains a sum and a count from each worker,
    and then the coordinator node computes the final average.
+
+   Full list of the special-case aggregates:
+
+     invalid, avg, min, max, sum, count, array_agg, jsonb_agg,
```

Comment:
Not `invalid`, that's a bit of a silly string, I actually had to make a PR https://github.com/citusdata/citus/pull/2957 to fix us trying to distribute aggregates named `invalid` in release builds